example_project/example_project_combined.json

```diff
@@ -790,6 +790,7 @@
         "name": "Residential 4",
         "type": "Building",
         "building_type": "Single-Family Attached",
+        "number_of_stories": 2,
```

Comment:
This is pretty confusing. It's required but we don't validate this feature against the schema?

Reply:
I know...but for now we do need it for this process here: https://github.com/urbanopt/urbanopt-geojson-gem/blob/develop/lib/urbanopt/geojson/helper.rb#L284.  I will make a note to review this process to see if we could modify it or extract # stories another way, but for now it needs #stories...